lib/Elastica/Bulk/Action/AbstractDocument.php

```diff
@@ -9,20 +9,20 @@
 abstract class AbstractDocument extends Action
 {
     /**
-     * @var \Elastica\Document|\Elastica\Script\AbstractScript
+     * @var Document|AbstractScript
      */
     protected $_data;
 
     /**
-     * @param \Elastica\Document|\Elastica\Script\AbstractScript $document
+     * @param Document|AbstractScript $document
```

Comment:
I don't understand the question. `Elastica\Document` is already imported via use statement.